### ISSUE TITLE
Elements on documents explorer's tree disappear

### ISSUE THREAD

**Issue by AlexSerrano22** [CarbonLDP/carbonldp-workbench#56]
I saw that if you expand an element on the document's tree, and after that, you double-clicking the element that you expand, this element disappear.
I attach the images: 
<img width="1087" alt="screen shot 2018-01-17 at 10 18 11 pm" src="https://user-images.githubusercontent.com/5456395/35080769-4a5b4000-fbd5-11e7-8bde-eeac223eaa64.png">
<img width="1100" alt="screen shot 2018-01-17 at 10 18 31 pm" src="https://user-images.githubusercontent.com/5456395/35080770-4a765444-fbd5-11e7-89f2-c30812564ab1.png">

**Comment by alvaro9210** [comment]
I've confirmed the behavior. It seems that when a container queries for its children, if you double click it when it's making the request and the request hasn't finished, the node disappears causing an error with the tree library.
![image](https://user-images.githubusercontent.com/4240006/35081561-8a9b6d52-fbda-11e7-8098-0e4d191424cf.png)
Thanks for reporting the issue!!

**Comment by alvaro9210** [comment]
Adding extra console information for debugging purposes:
```
app.43679dca33355c574d85.js:1 ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'state' of undefined
TypeError: Cannot read property 'state' of undefined
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:formatted:12922)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:formatted:16513)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:formatted:12950)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:formatted:16513)
    at e.jstree.plugins.wholerow.delete_node (0.63dc1e304e246417972e.chunk.js:formatted:13721)
    at DocumentTreeViewComponent.emptyNode (0.63dc1e304e246417972e.chunk.js:formatted:19721)
    at 0.63dc1e304e246417972e.chunk.js:formatted:19692
    at ZoneDelegate.invoke (polyfills.ca4447d0991a01fbd3bd.js:1)
    at Object.onInvoke (app.43679dca33355c574d85.js:1)
    at ZoneDelegate.invoke (polyfills.ca4447d0991a01fbd3bd.js:1)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:formatted:12922)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:formatted:16513)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:formatted:12950)
    at e.jstree.plugins.wholerow.redraw_node (0.63dc1e304e246417972e.chunk.js:formatted:16513)
    at e.jstree.plugins.wholerow.delete_node (0.63dc1e304e246417972e.chunk.js:formatted:13721)
    at DocumentTreeViewComponent.emptyNode (0.63dc1e304e246417972e.chunk.js:formatted:19721)
    at 0.63dc1e304e246417972e.chunk.js:formatted:19692
    at ZoneDelegate.invoke (polyfills.ca4447d0991a01fbd3bd.js:1)
    at Object.onInvoke (app.43679dca33355c574d85.js:1)
    at ZoneDelegate.invoke (polyfills.ca4447d0991a01fbd3bd.js:1)
    at resolvePromise (polyfills.ca4447d0991a01fbd3bd.js:1)
    at resolvePromise (polyfills.ca4447d0991a01fbd3bd.js:1)
    at polyfills.ca4447d0991a01fbd3bd.js:1
    at ZoneDelegate.invokeTask (polyfills.ca4447d0991a01fbd3bd.js:1)
    at Object.onInvokeTask (app.43679dca33355c574d85.js:1)
    at ZoneDelegate.invokeTask (polyfills.ca4447d0991a01fbd3bd.js:1)
    at Zone.runTask (polyfills.ca4447d0991a01fbd3bd.js:1)
    at drainMicroTaskQueue (polyfills.ca4447d0991a01fbd3bd.js:1)
    at ZoneTask.invokeTask [as invoke] (polyfills.ca4447d0991a01fbd3bd.js:1)
    at p (polyfills.ca4447d0991a01fbd3bd.js:1)
  | defaultErrorLogger | @ | app.43679dc….js:1
  | ErrorHandler.handleError | @ | app.43679dc….js:1
  | next | @ | app.43679dc….js:1
  | t.object.o | @ | app.43679dc….js:1
  | SafeSubscriber.__tryOrUnsub | @ | app.43679dc….js:1
  | SafeSubscriber.next | @ | app.43679dc….js:1
  | Subscriber._next | @ | app.43679dc….js:1
  | Subscriber.next | @ | app.43679dc….js:1
  | Subject.next | @ | app.43679dc….js:1
  | EventEmitter.emit | @ | app.43679dc….js:1
  | (anonymous) | @ | app.43679dc….js:1
  | ZoneDelegate.invoke | @ | polyfills.ca4447d….js:1
  | Zone.run | @ | polyfills.ca4447d….js:1
  | NgZone.runOutsideAngular | @ | app.43679dc….js:1
  | onHandleError | @ | app.43679dc….js:1
  | ZoneDelegate.handleError | @ | polyfills.ca4447d….js:1
  | Zone.runGuarded | @ | polyfills.ca4447d….js:1
  | (anonymous) | @ | polyfills.ca4447d….js:1
  | n.microtaskDrainDone | @ | polyfills.ca4447d….js:1
  | drainMicroTaskQueue | @ | polyfills.ca4447d….js:1
  | ZoneTask.invokeTask | @ | polyfills.ca4447d….js:1
  | p | @ | polyfills.ca4447d….js:1
  | h | @ | polyfills.ca4447d….js:1
  | XMLHttpRequest.send (async) |   |  
  | scheduleTask | @ | polyfills.ca4447d….js:1
  | ZoneDelegate.scheduleTask | @ | polyfills.ca4447d….js:1
  | onScheduleTask | @ | polyfills.ca4447d….js:1
  | ZoneDelegate.scheduleTask | @ | polyfills.ca4447d….js:1
  | Zone.scheduleTask | @ | polyfills.ca4447d….js:1
  | Zone.scheduleMacroTask | @ | polyfills.ca4447d….js:1
  | (anonymous) | @ | polyfills.ca4447d….js:1
  | o.(anonymous function) | @ | polyfills.ca4447d….js:1
  | (anonymous) | @ | app.43679dc….js:1
  | ZoneAwarePromise | @ | polyfills.ca4447d….js:1
  | sendWithBrowser | @ | app.43679dc….js:1
  | Service.send | @ | app.43679dc….js:1
  | Service.post | @ | app.43679dc….js:1
  | Class.executeRawSELECTQuery | @ | app.43679dc….js:1
  | Class.executeSELECTQuery | @ | app.43679dc….js:1
  | (anonymous) | @ | app.43679dc….js:1
  | (anonymous) | @ | app.43679dc….js:1
  | ZoneAwarePromise | @ | polyfills.ca4447d….js:1
  | promiseMethod | @ | app.43679dc….js:1
  | Class.executeSELECTQuery | @ | app.43679dc….js:1
  | DocumentTreeViewComponent.getNodeChildren | @ | 0.63dc1e304e24641797….js:formatted:19729
  | DocumentTreeViewComponent.onBeforeOpenNode | @ | 0.63dc1e304e24641797….js:formatted:19691
  | (anonymous) | @ | 0.63dc1e304e24641797….js:formatted:19659
  | dispatch | @ | app.43679dc….js:1
  | C.handle | @ | app.43679dc….js:1
  | trigger | @ | app.43679dc….js:1
  | triggerHandler | @ | app.43679dc….js:1
  | trigger | @ | 0.63dc1e304e24641797….js:formatted:12054
  | open_node | @ | 0.63dc1e304e24641797….js:formatted:12994
  | toggle_node | @ | 0.63dc1e304e24641797….js:formatted:13073
  | (anonymous) | @ | 0.63dc1e304e24641797….js:formatted:11886
  | i | @ | app.43679dc….js:1
  | dispatch | @ | app.43679dc….js:1
  | C.handle | @ | app.43679dc….js:1
  | ZoneDelegate.invokeTask | @ | polyfills.ca4447d….js:1
  | onInvokeTask | @ | app.43679dc….js:1
  | ZoneDelegate.invokeTask | @ | polyfills.ca4447d….js:1
  | Zone.runTask | @ | polyfills.ca4447d….js:1
  | ZoneTask.invokeTask | @ | polyfills.ca4447d….js:1
  | p | @ | polyfills.ca4447d….js:1
  | h | @ | polyfills.ca4447d….js:1
```

**Comment by alvaro9210** [comment]
This has been solved in `v1.0.0-alpha.3` of the workbench. Please update.
Closing #57 too.
